### ISSUE TITLE
Support @Value Annotation

### ISSUE THREAD

**Issue by emeroad** [pinpoint-apm/pinpoint#7551]
Add spring-like `@Value` Annotation

```java
public class Configuration {
   @Value("${port}");
   private int port;
}
```

**Comment by emeroad** [comment]
Doesn't work in Java 9+
Violation of module accessibility
> Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private long com.navercorp.pinpoint.grpc.client.ClientOption.keepAliveTime accessible: module pinpoint.agent does not "opens com.navercorp.pinpoint.grpc.client" to unnamed module @41629346
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:361)
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:301)
	at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:177)
	at java.base/java.lang.reflect.Field.setAccessible(Field.java:171)
	at com.navercorp.pinpoint.bootstrap.config.util.ValueAnnotationProcessor.setAccessible(ValueAnnotationProcessor.java:176)
	at com.navercorp.pinpoint.bootstrap.config.util.ValueAnnotationProcessor.injectField(ValueAnnotationProcessor.java:214)
	at com.navercorp.pinpoint.bootstrap.config.util.ValueAnnotationProcessor.handleFields(ValueAnnotationProcessor.java:76)
	at com.navercorp.pinpoint.bootstrap.config.util.ValueAnnotationProcessor.process(ValueAnnotationProcessor.java:67)
	at pinpoint.agent/pinpoint.agent/com.navercorp.pinpoint.profiler.context.grpc.GrpcTransportConfig.readClientOption(GrpcTransportConfig.java:170)
	at pinpoint.agent/pinpoint.agent/com.navercorp.pinpoint.profiler.context.grpc.GrpcTransportConfig.readAgentClientOption(GrpcTransportConfig.java:150)
	at pinpoint.agent/pinpoint.agent/com.navercorp.pinpoint.profiler.context.grpc.GrpcTransportConfig.read(GrpcTransportConfig.java:137)
	at pinpoint.agent/pinpoint.agent/com.navercorp.pinpoint.profiler.context.module.GrpcModule.loadGrpcTransportConfig(GrpcModule.java:145)
	at pinpoint.agent/pinpoint.agent/com.navercorp.pinpoint.profiler.context.module.GrpcModule.configure(GrpcModule.java:78)
	at pinpoint.agent/pinpoint.agent/com.google.inject.PrivateModule.configure(PrivateModule.java:100)
	at pinpoint.agent/pinpoint.agent/com.google.inject.spi.Elements$RecordingBinder.install(Elements.java:340)
	at pinpoint.agent/pinpoint.agent/com.google.inject.util.Modules$CombinedModule.configure(Modules.java:128)
	at pinpoint.agent/pinpoint.agent/com.google.inject.spi.Elements$RecordingBinder.install(Elements.java:340)
	at pinpoint.agent/pinpoint.agent/com.google.inject.spi.Elements.getElements(Elements.java:110)
	at pinpoint.agent/pinpoint.agent/com.google.inject.internal.InjectorShell$Builder.build(InjectorShell.java:138)
	at pinpoint.agent/pinpoint.agent/com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:104)
	... 21 more
